upup/pkg/fi/cloudup/docker_test.go

```diff
@@ -38,44 +38,58 @@ func TestDockerVersionUrlHash(t *testing.T) {
 	}{
 		{
 			arch:    architectures.ArchitectureAmd64,
-			version: "19.03.13",
-			url:     "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.13.tgz",
-			hash:    "ddb13aff1fcdcceb710bf71a210169b9c1abfd7420eeaf42cf7975f8fae2fcc8",
+			version: "17.12.1",
+			url:     "https://download.docker.com/linux/static/stable/x86_64/docker-17.12.1-ce.tgz",
+			hash:    "1270dce1bd7e1838d62ae21d2505d87f16efc1d9074645571daaefdfd0c14054",
```

Comment:
This file should not change, it contains just unit tests and there is no benefit in updating it.